compiler/rustc_middle/src/query/mod.rs

```diff
@@ -2490,7 +2490,12 @@ rustc_queries! {
         desc { "functions to skip for move-size check" }
     }
 
-    query items_of_instance(key: (ty::Instance<'tcx>, CollectionMode)) -> (&'tcx [Spanned<MonoItem<'tcx>>], &'tcx [Spanned<MonoItem<'tcx>>]) {
+    query items_of_instance(
```

Comment:
Sorry, I know I said it's fine to change this here, but only because I thought it was necessary to fix the issue. If this is no longer necessary, please move it to another PR. :pray: